sql/default.sql

```diff
@@ -4,3 +4,15 @@ CREATE TABLE IF NOT EXISTS expiredTeam(
 	teamID INTEGER NOT NULL,
 	created_at INTEGER DEFAULT (datetime('now', 'localtime'))
 );
+
+CREATE TABLE IF NOT EXISTS evaluators(
```

Comment:
I like this! Perhaps later we can use this to cache users as well and reduce the amount API calls.